gnus-recent.el

```diff
@@ -47,92 +47,143 @@
 
 (require 'gnus-sum)
 (require 'org-gnus)
+(require 'rfc2047)
+(require 'helm-lib)
+(require 'bbdb)
+(require 'bbdb-mua)
```

Comment:
The bbdb dependency seems OK to have in gnus-recent.el (would be kind of hard to do otherwise without inventing a plugin system and completely over-engineer this to death), but I don't see it included in my emacs 26, so that needs to go in Package-Requires then. 

(An alternative would be to do `(require 'bbdb nil 'noerror)` and make it an optional dependency, but I think we can leave that until someone who actually uses gnus without bbdb comes along and asks for it ;))

Reply:
best approach probably is to autoload the bbdb functions. Agree leave it, until someone asks.